packages/machine/src/middleware/state-transition/install-proposer.ts

```diff
@@ -77,11 +75,11 @@ export class InstallProposer {
     context: Context,
     data: cf.legacy.app.InstallData
   ): string[] {
-    const lastResult = getLastResult(Opcode.IO_WAIT, context.results);
+    const lastResult = context.intermediateResults.inbox!;
```

Comment:
@IIIIllllIIIIllllIIIIllllIIIIllllIIIIll In which context would `inbox` be unavailable? And why do we need to silent the compiler here?